client/src/ui/deckbuilder/DeckBuilderMain.vue

```diff
@@ -1,11 +1,19 @@
 <template>
   <div class="_deck-builder-main">
+    <DeckBuilderSectionControls
+        :deckIndex="store.selectedSeat"
```

Comment:
It looks like DeckBuilderSectionControls already imports the vuex store, so no need to pass the deckIndex here; it can just read it out of the store directly (if you need to reference the store in an html template, just create a computed property named `store` that returns the store).